runners-template-vms-creating/create_centos6.8_vm.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -e
+
+ostype='RedHat_64'
+disk_space='32768' # MB
+memory='8192'      # MB
+cpus='2'
+iso='CentOS-6.8-x86_64-minimal'
+storage_sata='SATA Controller'
+storage_ide='IDE Controller'
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+usage() {
+  cat <<EOF
+  usage: $0 -n VM_NAME [-r]
+  This script creates
+    1) an empty VM with CentOS 6.8 ready to be installed, or
+    2) CentOS 6.8 VM when -r parameter is applied
+       (reuses vdi named VM_NAME located in SCRIPT_DIR/vdis).
+  OPTIONS:
+    -h Show this message
+    -n Provide a name for VM
+    -r Reuses a vdi that already exists
+EOF
+}
+
+create_empty_vm() {
+  VBoxManage createvm --name "$vm_name" --ostype "$ostype" --register
+
+  local vm_info=$(VBoxManage showvminfo "$vm_name")
```

Comment:
Does this return a string without spaces always? if the answer is no or not known please add quotes around "$(...)"
Same for the lines below.